### ISSUE TITLE
More clear and detailed helps for commands

### ISSUE THREAD

**Issue by double-fault** [Bone-Project/BoneOS#158]
We need more detailed help for commands. This is going to be solved by some of the commands planned to be implemented in the "New Commands" issue, but making this separate issue so that people can give their views if they need some better help in some specific command.

These are the commands [mentioned in the new commands issue](https://github.com/Bone-Project/BoneOS/issues/125#issue-198234178) which will give better documentation:

 - man
 - whatis
 - tldr

---

This issue does not ask for new help commands, those belong to the new commands issue (please post them there, I will be happy to implement them). This is asking for your views on these commands and how they can be made even better.

**Comment by amanuel2** [comment]
i have never seen tldr command.. Seems very creative and actually pretty useful to the community for a short simple description.
